src/OmniSharp.MSBuild/ProjectLoader.cs

```diff
@@ -127,10 +131,7 @@ private static void SetTargetFrameworkIfNeeded(MSB.Evaluation.Project evaluatedP
                 // do better and potentially allow OmniSharp hosts to select a target framework.
                 targetFramework = targetFrameworks[0];
```

Comment:
how could we allow this to be overrideable in the future? 
allow the user to set `preferred TFM` via `MSBuildOptions` in the config?

Reply:
We need create a singleton service or something that we can toggle the tfm for a given project, and then also enhance services to also understand that...

Reply:
Yes, we need the same thing for setting the active solution/project configuration as well.